e2e/hiddenTokensFlow.spec.js

```diff
@@ -22,8 +33,10 @@ describe('Hidden tokens flow', () => {
     await Helpers.waitAndTap('wrapped-nft-rainbowtestwallet.eth');
     await Helpers.waitAndTap('unique-token-expanded-state-context-menu-button');
 
-    await waitFor(element(by.label('Hide')).atIndex(1)).toBeVisible();
-    await element(by.label('Hide')).atIndex(1).tap();
+    await waitFor(element(by.text('Hide')))
+      .toBeVisible()
+      .withTimeout(2000);
+    await element(by.text('Hide')).tap();
```

Comment:
Note to self: probably another helper like waitAndTap, but variant that looks by text instead of ID